src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -0,0 +1,129 @@
+    skeliingtonCase.setAddressLine3(collectionCase.getAddress().getAddressLine3());
+    skeliingtonCase.setTownName(collectionCase.getAddress().getTownName());
+    skeliingtonCase.setPostcode(collectionCase.getAddress().getPostcode());
+    skeliingtonCase.setArid(collectionCase.getAddress().getArid());
+    skeliingtonCase.setLatitude(collectionCase.getAddress().getLatitude());
+    skeliingtonCase.setLongitude(collectionCase.getAddress().getLongitude());
+    skeliingtonCase.setUprn(collectionCase.getAddress().getUprn());
+    skeliingtonCase.setRegion(collectionCase.getAddress().getRegion());
+    skeliingtonCase.setActionPlanId(collectionCase.getActionPlanId()); // This is essential
+    skeliingtonCase.setTreatmentCode(collectionCase.getTreatmentCode()); // This is essential
+    skeliingtonCase.setAddressLevel(collectionCase.getAddress().getAddressLevel());
+    skeliingtonCase.setAbpCode(collectionCase.getAddress().getApbCode());
+    skeliingtonCase.setCaseType(collectionCase.getCaseType());
+    skeliingtonCase.setAddressType(collectionCase.getAddress().getAddressType());
+    skeliingtonCase.setUprn(collectionCase.getAddress().getUprn());
+    skeliingtonCase.setEstabArid(collectionCase.getAddress().getEstabArid());
+    skeliingtonCase.setEstabType(collectionCase.getAddress().getEstabType());
+    skeliingtonCase.setOrganisationName(collectionCase.getAddress().getOrganisationName());
+    skeliingtonCase.setOa(collectionCase.getOa());
+    skeliingtonCase.setLsoa(collectionCase.getLsoa());
+    skeliingtonCase.setMsoa(collectionCase.getMsoa());
+    skeliingtonCase.setLad(collectionCase.getLad());
+    skeliingtonCase.setHtcWillingness(collectionCase.getHtcWillingness());
+    skeliingtonCase.setHtcDigital(collectionCase.getHtcDigital());
+    skeliingtonCase.setFieldCoordinatorId(collectionCase.getFieldCoordinatorId());
+    skeliingtonCase.setFieldOfficerId(collectionCase.getFieldOfficerId());
+    skeliingtonCase.setCeExpectedCapacity(collectionCase.getCeExpectedCapacity());
+    skeliingtonCase.setCeActualResponses(collectionCase.getCeActualResponses());
+    skeliingtonCase.setHandDelivery(collectionCase.isHandDelivery());
+
+    skeliingtonCase.setSurvey("CENSUS");
+    skeliingtonCase.setRefusalReceived(false);
+    skeliingtonCase.setReceiptReceived(false);
+    skeliingtonCase.setAddressInvalid(false);
+    skeliingtonCase.setUndeliveredAsAddressed(false);
+    return skeliingtonCase;
+  }
+
+  // https://collaborate2.ons.gov.uk/confluence/display/SDC/Handle+New+Address+Reported+Events
+  // Only a small number of mandatory fields to create a skellington case
+  private UUID validDateNewAddressCollectionCaseForMandatoryFields(
+      CollectionCase newCollectionCase) {
+    UUID newCaseId;
+
+    try {
+      newCaseId = UUID.fromString(newCollectionCase.getId());
+    } catch (IllegalArgumentException e) {
+      throw new RuntimeException(
```

Comment:
why have we done this when we'll already get a RTE? We made a decision not to bloat our codebase with loads of code like this.